test/__tests__/line-after-filepath-with-errors.js

```diff
@@ -0,0 +1,44 @@
+import { runCli } from "../utils";
+import { color } from "specialist";
+import path from 'node:path';
+
+const FIXTURES_PATH = path.join(process.cwd(), "test", "__fixtures__");
+
+describe("Line breaking after filepath with errors", () => {
+  // TODO (43081j): this test doesn't log an error, but does in prettier.
+  // we should decide if the behaviour should match
+  runCli("syntax-errors", ["./*.{js,unknown}"]).test({
+    status: 1,
+    stderr: "",
+  });
+  // TODO (43081j): this test throws errors in prettier. here it doesn't, and
+  // lists each invalid file path in the output (stderr). we should decide
+  // if to align with prettier
```

Comment:
see this `TODO` and the one above

this has been raised in another PR too. basically this block:

https://github.com/prettier/prettier-cli/blob/5d5a6ca186ce332d1dba0177e5ddb73186803899/src/index.ts#L181-L185

means that with no args, errors are not raised and are ignored

this means running with `list: true` or `check: true` behaves differently than with no args (i.e. both `false`)

Reply:
I think we should first add the fourth mode of operation as mentioned in #30, then I need to look at what the difference is exactly, because if prettier just throws it seems better to output a comprehensible error, while still erroring, no?

Reply:
The "forth mode", "dump" is implemented now, maybe we can reference that here if needed.

Reply:
the two tests fail in prettier because the files matching the glob are invalid I think (syntax errors)

so in this test (`--list-different`) and the argless one, we do error in prettier but not in prettier CLI 

the fourth mode doesn't seem relevant now that I think about it. in general if a glob matches invalid files, it seems we skip them? but I don't remember what past-james was doing here, so I'm just going off the TODOs